README.md

```diff
@@ -84,48 +149,54 @@ ts.execute(Policy.PERFORMANCE);
-
+* Dynamic
+  reconfiguration: [https://dl.acm.org/doi/10.1145/3313808.3313819](https://dl.acm.org/doi/10.1145/3313808.3313819)
 
 ## 5. How to Use it in your Projects?
 
-You can import the API and start using TornadoVM. Set this in the `pom.xml` file. 
+You can import the API and start using TornadoVM. Set this in the `pom.xml` file.
 
 ```xml
-   <repositories>
-     <repository>
-       <id>universityOfManchester-graal</id>
-       <url>https://raw.githubusercontent.com/beehive-lab/tornado/maven-tornadovm</url>
-     </repository>
-   </repositories>
-  
-   <dependencies>   
-      <dependency>
-         <groupId>tornado</groupId>
-         <artifactId>tornado-api</artifactId>
-         <version>0.8</version>
-      </dependency>
-      <dependency>
-         <groupId>tornado</groupId>
-         <artifactId>tornado-matrices</artifactId>
-         <version>0.8</version>
-      </dependency>
-   </dependencies>
+
+<repositories>
+    <repository>
+        <id>universityOfManchester-graal</id>
+        <url>https://raw.githubusercontent.com/beehive-lab/tornado/maven-tornadovm</url>
+    </repository>
+</repositories>
+
+<dependencies>
+<dependency>
+    <groupId>tornado</groupId>
+    <artifactId>tornado-api</artifactId>
+    <version>0.8</version>
+</dependency>
+<dependency>
+    <groupId>tornado</groupId>
+    <artifactId>tornado-matrices</artifactId>
+    <version>0.8</version>
+</dependency>
+</dependencies>
 ```
 
-To run TornadoVM, you need to either install the TornadoVM extension for GraalVM/OpenJDK, or run with our docker [images](assembly/src/docs/12_INSTALL_WITH_DOCKER.md).
+To run TornadoVM, you need to either install the TornadoVM extension for GraalVM/OpenJDK, or run with our
+docker [images](assembly/src/docs/12_INSTALL_WITH_DOCKER.md).
 
 ## 6. Additional Resources
 
-[Here](assembly/src/docs/16_RESOURCES.md) you can find videos, presentations, and articles and artefacts describing TornadoVM and how to use it.
+[Here](assembly/src/docs/16_RESOURCES.md) you can find videos, presentations, and articles and artefacts describing
+TornadoVM and how to use it.
 
 ## 7. Academic Publications
 
 Selected publications and citations can be found [here](assembly/src/docs/14_PUBLICATIONS.md).
 
 ## 8. Acknowledgments
 
-This work was initially supported by the EPSRC grants [PAMELA EP/K008730/1](http://apt.cs.manchester.ac.uk/projects/PAMELA/) and AnyScale Apps EP/L000725/1, and now it is funded by the [EU Horizon 2020 E2Data 780245](https://e2data.eu) and the [EU Horizon 2020 ACTiCLOUD 732366](https://acticloud.eu) grants.
+This work was initially supported by the EPSRC
+grants [PAMELA EP/K008730/1](http://apt.cs.manchester.ac.uk/projects/PAMELA/) and AnyScale Apps EP/L000725/1, and now it
+is funded by the [EU Horizon 2020 E2Data 780245](https://e2data.eu) and
+the [EU Horizon 2020 ACTiCLOUD 732366](https://acticloud.eu) grants.
```

Comment:
let's also add the ELEGANT

Reply:
@kotselidis , do you have the EU numbers for ELEGANT?

Reply:
https://www.elegant-h2020.eu/ 957286

Reply:
I got it: https://cordis.europa.eu/project/id/957286